src/proto/thread_telemetry.proto

```diff
@@ -496,6 +496,14 @@ message TelemetryData {
     optional uint32 global_unicast_address_count = 7;
   }
 
+  message NodeDivergenceInfo {
+    // Indicate whether the BR's external route is published
```

Comment:
We may need some more details about what it means.

Reply:
added some more description

Reply:
Perhaps we should have 3 bool metrics: whether the route is publishing route for `::/0`(any network), `fc00::/7`(any ULA network), and other ULA prefix.

Reply:
updated, added a new dedicated message `ExternalRoutes` to show the information about external route, from which we can get the information about `::/0`, `fc00::/7` or others